ci-operator/jobs/openshift/ironic-agent-image/openshift-ironic-agent-image-main-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   openshift/ironic-agent-image:
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
Doesn't setting this to `false` mean the job will never run unless somebody explicitly puts a comment to `/test e2e-metal-ipi`?

Reply:
Ah, nope, according to https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#triggering-jobs-based-on-changes the semantics of `skip_if_only_changed` are closer to `run_if_not_only_changed`, so you are correct.